modules/osdk-hh-create-project.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * operators/operator_sdk/helm/osdk-hybrid-helm.adoc
+
+:_content-type: PROCEDURE
+[id="osdk-hh-create-project_{context}"]
+= Creating a project
+
+Use the Operator SDK CLI to create a project called `memcached-operator`.
+
+.Procedure
+
+. Create a directory for the project:
++
+[source,terminal]
+----
+$ mkdir -p $HOME/github.com/example/memcached-operator
+----
+
+. Change to the directory:
++
+[source,terminal]
+----
+$ cd $HOME/github.com/example/memcached-operator
+----
+
+. Run the `operator-sdk init` command to initialize the project. Use a domain of `example.com` so that all API groups are `<group>.example.com`:
```

Comment:
Where do you specify the domain?

Reply:
@emmajiafan @varshaprasad96 Is the [command below](https://github.com/openshift/openshift-docs/pull/41136/files#diff-170cf6994d784ced777fe811eee8b71ddc2213e63900222fdf74c535edf40ad3R31-R34) missing `--domain example.com`?

Reply:
Yes, the domain can be specified in the init command.